src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,62 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.UserAlreadyExistException;
+import core.basesyntax.exceptions.UserRegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int VALID_PASSWORD_LENGTH = 6;
+    private static final int VALID_LOGIN_LENGTH = 6;
+    private static final int VALID_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        isUserNull(user);
+        isValidUserLogin(user);
+        isValidUserPassword(user);
+        isValidUserAge(user);
```

Comment:
Such names are better suited for methods that return boolean values, for void methods better use validatePassword, validateLogin etc.